readme.md

```diff
@@ -26,6 +26,7 @@ is built from various Ubuntu starting images.
 | ansible | Pipx (Python 3.10) | 6.6.0 |
 | ansible-base | Pipx (Python 3.10) | 2.10.17 |
 | build-essential | apt-get | latest |
+| cargo-deny | cargo | latest |
```

Comment:
@NiklasRosenstein I replaced `cargo-audit` with `cargo-deny` because it is more flexible.

Like `cargo-audit`, `cargo-deny` can check dependencies for known vulnerabilities, but it includes additional functionalities that could be useful (like allowing/blocking dependencies by licence, name, version, etc).